CHANGELOG.rst

```diff
@@ -1,4 +1,4 @@
-**4.3.0 - 07/29/25**
+**4.3.0 - 07/30/25**
```

Comment:
as discussed, I'll keep this as a (presumably backwards-compatible) feature update and then we'll major-roll when we remove the imports.